assets/js/theme/global/modal.js

```diff
@@ -218,25 +230,33 @@ export class Modal {
 
     setupFocusableElements(modalType) {
         this.$preModalFocusedEl = $(document.activeElement);
+        const { $modalTabbableCollection, modalSelector } = focusableElements[modalType]();
 
-        const $collection = focusableElements[modalType]();
-        $collection.get(0).focus();
+        this.modalSelector = modalSelector;
```

Comment:
Thank you for changes, after that we don't need lane 235 as well

Reply:
thanks, working on it right now:)